### DISCUSSION TITLE
Zip file is damaged

### DISCUSSION THREAD

**Discussion by AJGestel** [onlaj/Piano-LED-Visualizer#504] [General]
Zip file is damaged

It no longer has any content. I can't use it.
He is also very small now.

**Comment by onlaj** [comment]
Hello.
Which zip file? I checked the latest one and it seems fine. Try different app for unzipping.

**Reply by AJGestel** [reply]
Piano-LED-Visualizer-master.
It is only 1.68Mb
[Piano-LED-Visualizer-master.zip](https://github.com/onlaj/Piano-LED-Visualizer/files/14904300/Piano-LED-Visualizer-master.zip)

**Reply by onlaj** [reply]
Download image, not a source code.
https://github.com/onlaj/Piano-LED-Visualizer/releases/download/v1.5.1/PLV_v1.5.1.zip

**Reply by AJGestel** [reply]
Ok.
Thank you.
